### PR TITLE
Tests zarr methods part3

### PR DESCRIPTION
**Pull request by chrisrossi**
Note that the basis for the diff is #117

### CODE REVIEW

File: gridded_etl_tools/utils/zarr_methods.py [hunk 1]
```diff
@@ -420,8 +423,8 @@ def convert_to_lowest_common_time_denom(self, raw_files: list, keep_originals: b
         keep_originals : bool, optional
             An optional flag to preserve the original files for debugging purposes. Defaults to False.
         """
-        if len(list(raw_files)) == 0:
-            raise FileNotFoundError("No files found to convert, exiting script")
+        if len(raw_files) == 0:
+            raise FileNotFoundError("No files found to convert, exiting script")  # Seems like more of a ValueError
```
Comment (rbanick) [inline gridded_etl_tools/utils/zarr_methods.py:427]: I’m down to change

File: gridded_etl_tools/utils/zarr_methods.py [hunk 2]
```diff
@@ -539,6 +543,7 @@ def parse(self, *args, **kwargs) -> bool:
         if hasattr(self, "dataset_hash") and self.dataset_hash and not self.dry_run:
             self.info("Published dataset's IPFS hash is " + str(self.dataset_hash))
 
+        # TODO: Is anything relying on this return value? If return value is invariant, may as well not return a value.
```
Comment (rbanick) [inline gridded_etl_tools/utils/zarr_methods.py:546]: We can nix the boolean but I’d like to return a message of some sort indicating success. That’s of minor utility during development and testing.

File: gridded_etl_tools/utils/zarr_methods.py [hunk 3]
```diff
@@ -347,7 +348,7 @@ def preprocess_kerchunk(cls, refs: dict) -> dict:
                 ref_names.add(re.match(file_match_pattern, ref).group(1))
         for ref in ref_names:
             fill_value_fix = json.loads(refs[f"{ref}/.zarray"])
-            fill_value_fix["fill_value"] = str(cls.missing_value)
+            fill_value_fix["fill_value"] = str(cls.missing_value)  # TODO: Do we need `str`? Gets dumped to json below
```
Comment (rbanick) [inline gridded_etl_tools/utils/zarr_methods.py:351]: Again, I agree this `str` is superfluous

**Review by rbanick — APPROVED** [review]

File: gridded_etl_tools/utils/zarr_methods.py (hunk 2 above)
Comment (chrisrossi) [reply gridded_etl_tools/utils/zarr_methods.py:546]: Return or log?

Comment (rbanick) [reply gridded_etl_tools/utils/zarr_methods.py:546]: I’m fine with a log statement

File: gridded_etl_tools/utils/zarr_methods.py (hunk 1 above)
Comment (chrisrossi) [reply gridded_etl_tools/utils/zarr_methods.py]: 4b82f85

File: gridded_etl_tools/utils/zarr_methods.py (hunk 3 above)
Comment (chrisrossi) [reply gridded_etl_tools/utils/zarr_methods.py]: 4b82f85

**Pull request merged**